packages/shared/styles/components/molecules/SfSteps.scss

```diff
@@ -41,22 +42,41 @@
     left: 0;
     bottom: var(--steps-progress-bottom, -0.09375rem);
     display: block;
-    height: var(--steps-progress-height, 0.1875rem);
-    background: var(--steps-progress-background, var(--c-dark));
+    height: var(--steps-progress-height, 0.188rem);
+    background: var(--steps-progress-background, var(--c-primary));
     transform-origin: 0 50%;
     transition: var(--steps-progress-transition, transform 150ms ease-in-out);
   }
   &__title {
-    position: relative;
-    display: block;
     text-align: center;
   }
-  &__content {
-    padding: var(--steps-content-padding, 2.5rem 0 0 0);
-  }
   @include for-desktop {
-    --steps-progress-height: 0.125rem;
-    --steps-content-padding: 5rem 0 0 0;
-    --steps-progress-bottom: -0.09375rem;
+    --steps-step-color: var(--c-text-muted);
+    &__header {
+      box-shadow: none;
+      @include border(--steps-border, 0 0 0.125rem 0, solid, var(--c-light));
```

Comment:
here you should just modify variable, you should set it in mobile view and set 0 for width
```
      @include border(--steps-border, 0, solid, var(--c-light));
```